### PR TITLE
kevin/footer

### LINKED ISSUE #19: Footer
- import the logo at the top
- use the ICONS map in src/data/icons.js to display icon based on name
image

### PR DESCRIPTION
**Pull request by Kevinloritsch**

### CODE REVIEW

**Comment by Kevinloritsch** [comment]
image

File: src/components/Footer.jsx [hunk 1]
```diff
@@ -1,5 +1,52 @@
...
+import { ICONS } from "@/data/icons";
+
 const Footer = () => {
-  return <div>Footer</div>;
+  return (
+    <div className="w-full bg-ewb-green">
+      <p className="leading-4"> &nbsp; </p>
+
+      <div className="w-full bg-ewb-blue-200 border-t-8 border-ewb-green">
+        <br />
```
Comment (menthy-wu) [inline src/components/Footer.jsx]: instead of br, use patting top in classname

File: src/components/Footer.jsx [hunk 2]
```diff
@@ -1,5 +1,52 @@
...
+import Image from "next/image";
+import { ICONS } from "@/data/icons";
+
 const Footer = () => {
-  return <div>Footer</div>;
+  return (
+    <div className="w-full bg-ewb-green">
+      <p className="leading-4"> &nbsp; </p>
+
+      <div className="w-full bg-ewb-blue-200 border-t-8 border-ewb-green">
```
Comment (menthy-wu) [inline src/components/Footer.jsx]: make this the parent component.
think of the entire footer as two parts: the logo at the left and the information at the right. You can add a "flex justify-between at the parent component

File: src/components/Footer.jsx [hunk 3]
```diff
@@ -1,5 +1,52 @@
...
+        <br />
+        <br />
+        <br />
+        <br />
+        <br />
+        <table className="w-full">
```
Comment (menthy-wu) [inline src/components/Footer.jsx]: try use the justify-between instead of table

File: src/components/Footer.jsx [hunk 4]
```diff
@@ -1,5 +1,52 @@
...
+          <tbody>
+            <tr>
+              <td className="w-1/2">
+                <div className="mt-6 mx-6">
+                  <div className="inline-flex">
```
Comment (menthy-wu) [inline src/components/Footer.jsx]: reduce the amount of div wrappers, you can put all classNames in one div

**Review by menthy-wu — CHANGES_REQUESTED** [review]

**Comment by Kevinloritsch** [comment]
image

File: src/components/Footer.jsx [hunk 5]
```diff
@@ -1,5 +1,33 @@
+import whiteLogo from "../../public/assets/whiteLogo.svg";
+import Image from "next/image";
+import { ICONS } from "@/data/icons";
+
 const Footer = () => {
-  return <div>Footer</div>;
+  return (
+    <div className="w-full bg-ewb-green">
```
Comment (menthy-wu) [inline src/components/Footer.jsx]: remove this div

File: src/components/Footer.jsx [hunk 6]
```diff
@@ -1,5 +1,33 @@
...
+      <p className="leading-4"> &nbsp; </p>
+
+      <div className="w-full bg-ewb-blue-200 border-t-8 border-t-ewb-green pt-32 flex justify-between">
```
Comment (menthy-wu) [inline src/components/Footer.jsx]: border-t-[20px]

File: src/components/Footer.jsx [hunk 7]
```diff
@@ -1,5 +1,33 @@
...
+        <div className="inline-flex pl-6 pt-10 text-white font-bold text-xl">
+          <Image src={whiteLogo} alt="EWB Logo" className="scale-125" />
+          <p className="pt-16 pl-4">EWB at UCR</p>
+        </div>
+
+        <div className="px-1 text-right text-white m-3 text-sm">
+          <div className="ml-3 scale-125 float-right">
+            {" "}
```
Comment (menthy-wu) [inline src/components/Footer.jsx]: remove {" "}

**Review by menthy-wu — CHANGES_REQUESTED** [review]

**Comment by Kevinloritsch** [comment]
image

File: src/components/Footer.jsx [hunk 8]
```diff
@@ -1,5 +1,26 @@
...
-  return <div>Footer</div>;
+  return (
+    <div className="w-full bg-ewb-blue-200 border-t-[20px] border-t-ewb-green pt-32 flex justify-between">
+      <div className="inline-flex pl-6 pt-10 text-white font-bold text-xl">
+        <Image src={whiteLogo} alt="EWB Logo" className="scale-125" />
+        <p className="pt-16 pl-4">EWB at UCR</p>
+      </div>
+
+      <div className="px-1 text-right text-white m-3 text-sm">
+        <div className="ml-3 scale-125 float-right">{ICONS["facebook"]}</div>
```
Comment (menthy-wu) [inline src/components/Footer.jsx]: make it a link https://github.com/acm-ucr/ewb-website/blob/dev/src/components/home/Social.jsx

**Review by menthy-wu — CHANGES_REQUESTED** [review]

**Comment by Kevinloritsch** [comment]
image

Added link functionality.

**Review by menthy-wu — APPROVED** [review]

**Pull request merged**